### PR TITLE
Provide Feedback fix: Made it so fields are hidden when they should be

### LINKED ISSUE #316: Providing Feedback "Broken?" field does not disappear once toggling back
When on Provide Feedback modal/form, if you select "yes" you Did run the notebook and then toggle back and forth between the "Did it work?" field from "Yes" to "No", you will see the "Broken?" field that pops up doesn't disappear.

### PR DESCRIPTION
**Pull request by manfromjupyter**
Closes #316

### CODE REVIEW

**Comment by mcrutch** [comment]
HTML and plain text emails are inconsistent:
HTML:
"xxxx provided the following feedback: "This is feedback""
Plain Text:
"xxxx provided the following feedback: This is feedback Additionally, the user indicated that the notebook was run and it was not successful. The user provided the following feedback about what went wrong for them: I am going to say it worked, but leave this filled out"

Workflow; Click Ran it, click it didn't work, fill out the comment for not working, then change the checkbox to it did work.

**Review by mcrutch — CHANGES_REQUESTED** [review]
Server-side code is inconsistent with how it handles the checkbox being toggled back to Yes it worked after filling in the comment between plain text and HTML emails

**Comment by manfromjupyter** [comment]
@mcrutch, I'm on it. Ill be sure to fix those emails

**Comment by manfromjupyter** [comment]
Made them consistent and while was at it improved the verbiage, spacing, etc. Tested for when notebook was not run, run and no mention of broken, run and labeled broken, and run and labeled as not broken. Good to go on my end

File: app/views/notebook_mailer/feedback.html.slim [hunk 1]
```diff
@@ -11,9 +11,10 @@ p
     ==@feedback.general_feedback
     | "
 br
-- if @feedback.worked == false
-  p
-    | Additionally, the user indicated that the notebook was run and it was not successful.
+-if @feedback.worked == true
```
Comment (mcrutch) [inline app/views/notebook_mailer/feedback.html.slim:14]: You need to check if ran is true as well.
Workflow: Check that you ran it
Check that it didn't work
add feedback for how it didn't work
check that you didn't run it
submit feedback
Email says they reported it didn't work, even though that option was no longer visible on the form.

**Review by mcrutch — CHANGES_REQUESTED** [review]
See comment in code, applies to both text and HTML emails.

**Comment by mcrutch** [comment]
Forgot one other things...check if the org is actually populated before you add the (org) to the email (since you are fixing the views anyway).
    | #{@submitter.name} (#{@submitter.org}) <-- If org is empty, it says `User's Name ()`

**Comment by manfromjupyter** [comment]
@mcrutch Oh smart, will check for org and create another ticket to correct it for all others. Won't take me terribly long to do. As per the running thing, that is wacky. Will fix it though

**Comment by manfromjupyter** [comment]
@mcrutch Should be good to go now. Will fix orgs in other emails later. In terms of the different options when submitting feedback, below will include what both html and raw look like and you can see what all of the other sub options look like in the top 5 emails in the fake email thing. Fixed that submission issue of broken and ran feedback when user switches later they didn't even run it.
html feedback email
text feedback email

**Pull request merged**